docs/src/main/sphinx/sql/create-function.md

```diff
@@ -0,0 +1,52 @@
+# CREATE FUNCTION
+
+## Synopsis
+
+```text
+CREATE [OR REPLACE] FUNCTION
+  routine_definition
+```
+
+## Description
+
+Create or replace a [](routine-catalog). The `routine_definition` is composed of
+the usage of [](/routines/function) and nested statements. The name of the
+routine must be fully qualified with catalog and schema location, unless the
+[default SQL routine storage catalog and
+schema](/admin/properties-sql-environment) are configured. The connector used in
+the catalog must support routine storage.
+
+The optional `OR REPLACE` clause causes the routine to be replaced if it already
+exists rather than raising an error.
+
+## Examples
+
+The following example creates the `meaning_of_life` routine in the `default`
+schema of the `example` catalog:
+
+```sql
+CREATE FUNCTION example.default.meaning_of_life()
```

Comment:
Add an argument to the routine. Maybe make this a `multiply_by_two` function

Reply:
I am going to leave this for now since it works better with the following shorter version .. and we got more examples anyway.